src/components/ChartContainer.jsx

```diff
@@ -217,14 +221,24 @@ export default class ChartContainer extends React.Component {
                                             config.yAxisLabel || ((config.charts.length > 1 ? '' : config.charts[0].y)) :
                                             config.xAxisLabel || config.x
                                     }
-                                    axisLabelComponent={<VictoryLabel dy={0} />}
+                                    axisLabelComponent={
+                                        <VictoryLabel
+                                            dy={(config.style ? !!config.style.xAxisTickAngle : false ) ? 50 : 0}
+                                            style={{
+                                                fill: config.style ?
+                                                    config.style.axisLabelColor :
+                                                    currentTheme.axis.style.axisLabel.fill,
+                                            }}
+                                        />
+                                    }
                                     tickFormat={horizontal ? null : this.xAxisTickFormat(xScale, config, isOrdinal, arr)}
                                     standalone={false}
                                     tickLabelComponent={
                                         <VictoryLabel
                                             angle={config.style ? config.style.xAxisTickAngle || 0 : 0}
                                             theme={currentTheme}
-                                            textAnchor={'middle'}
+                                            textAnchor={config.style ? config.style.xAxisTickAngle ? 'start' :
+                                                'middle' : 'middle'}
```

Comment:
shall we use brackets to make this more readable? ( when there are two ternary operators

Reply:
Fixed with https://github.com/wso2/react-vizgrammar/pull/207